### DISCUSSION TITLE
Move "+ New Tab" button from bottom to top

### DISCUSSION THREAD

**Discussion by ghost** [zen-browser/desktop#1767] [Ideas]
The opened tabs that you switch between, accumulate at top of sidebar. While, the button to open new one sits at the very bottom, this makes switching to new tabs painful.

**Comment by avoid-ashraful** [comment]
Is there a way to change the order of how new tabs are added in the browser's sidebar? Currently, when I open a new tab, it gets added to the bottom of the list. I’d prefer new tabs to open at the top of the sidebar instead. Does anyone know if this can be configured?

**Reply by barslmn** [reply]
Checking the show new tab button, and move new tab button to the top options, opens new tabs at top of the list for me.

![image](https://github.com/user-attachments/assets/6f2f63f4-c73b-47b8-b850-e4654a441d9d)

**Reply by Zydiag** [reply]
I think this is still not good for someone used to other browsers opening new tab next to existing one
you can use the following setting.

<img width="969" height="196" alt="image" src="https://github.com/user-attachments/assets/6b4f804e-5d13-4e85-a401-08636af7a56c" />
